### PR TITLE
beepsky replaces THREAT_LEVEL with the actual threat

### PR DESCRIPTION
**Pull request by timothyteakettle**
## About The Pull Request
beepsky replaces THREAT_LEVEL with the real threat level woops

## Why It's Good For The Game
i made an oopsy

## Changelog
:cl:
fix: beepsky replaces the word THREAT_LEVEL with the actual threat level
/:cl:

### CODE REVIEW

**Comment by timothyteakettle** [comment]
this doesnt fix it my bad let me try again

**Comment by timothyteakettle** [comment]
alright it now converts the threat number to text before checking it so it doesn't fail due to it being a threat level of 0

**Review by nik707 — APPROVED** [review]
